content/workshops/other.json

```
@@ -1,5 +1,30 @@
 {
     "events": [
+        {
+            "name": "Online CodeRefinery Workshop and Hackathon on Software Testing",
+            "website": "https://coderefinery.github.io/2021-03-17-testing-hackathon/",
+            "start_date": "2021-03-17",
+            "end_date": "2021-03-24",
+            "host": [
+                "Naoe Tatara"
+            ],
```

```suggestion
            "Thor Wikfeldt" ],
```
@wikfeldt I wonder if you are also acting as a host??